src/Arrays/vector_operations.jl

```diff
@@ -90,8 +91,47 @@ julia> samedir([1, 2, 3], [-1, -2, -3])
 
 ```
 """
-function samedir(u::AbstractVector{N},
-                 v::AbstractVector{N}) where {N<:Real}
+function samedir(u::AbstractVector{<:Real}, v::AbstractVector{<:Real})
+    return _ismultiple(u, v; allow_negative=false)
+end
+
+"""
+    ismultiple(u::AbstractVector{<:Real}, v::AbstractVector{<:Real})
+
+Check whether two vectors point are linearly dependent.
+
+### Input
+
+- `u` -- first vector
+- `v` -- second vector
+
+### Output
+
+`(true, k)` iff the vectors are identical up to a scaling factor `k ≠ 0`, and
```

Comment:
shall we specify that it is `u = kv` or `v = ku`

Reply:
yep, done